cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
```

Comment:
Which give us the example commands? We only should keep the used

Reply:
Understood, Cypress created this and I didn't remove, may be fine to remove

Reply:
so this file is required for the E2E testing in the event that a custom command is needed